### PR TITLE
adding quarterly review and planning description

### PR DESCRIPTION
**Pull request by dmurko**
Now in one issue: https://github.com/niteoweb/operations/issues/751

### CODE REVIEW

File: 2_Operations/work-process.md [hunk 1]
```diff
@@ -33,16 +33,16 @@ Our work process is based on [Scrum](https://en.wikipedia.org/wiki/Scrum_(softwa
...
-
 ## Alignment Meeting
 
 After the sprint retrospective we hold another meeting. Attendance is optional but every Nitean is welcome to participate. Primarily we look over the quarterly goals to determine if we are achieving them and to give us an idea of what should be included in the following day's sprint planning session.
 
 The main purpose of the alignment meeting is to adjust our compasses so that we will select the correct user stories in the sprint planning meeting coming the next day. We know we are paddling hard, but are we all paddling in the right direction?
 
+## Quarterly Review and Planning
+
+Every quarter we take time to review, reflect, and plan for the next quarter. There are three tasks for the Quarterly Review and Planning: Partners Meeting, Review, and All Hands Meeting.
```
Comment (karantan) [inline 2_Operations/work-process.md]: What is `Review`?

Comment (dmurko) [reply 2_Operations/work-process.md]: https://github.com/niteoweb/operations/issues/751

Screenshot 2019-03-13 at 16 00 43

Comment (karantan) [reply 2_Operations/work-process.md]: I would say `Quarterly Review` because just "Review" doesn't say a lot.

Comment (dmurko) [reply 2_Operations/work-process.md]: Fair point.

**Pull request merged**